examples/trusted_information_retrieval/proto/trusted_information_retrieval.proto

```diff
@@ -42,3 +42,41 @@ service TrustedInformationRetrieval {
   // Find the nearest Point Of Interest based on linear search in the database.
   rpc ListPointsOfInterest(ListPointsOfInterestRequest) returns (ListPointsOfInterestResponse);
 }
+
+message DatabaseEntry {
```

Comment:
Can this be renamed into `BikeStation`? `DatabaseEntry` sounds too generic for the field names in this message.

Reply:
In the next PR I'll remove specific fields from this message, and will just return a serialized string (in order to create a generic reusable module).